towhee/dag/graph_repr.py

```diff
@@ -21,16 +23,31 @@
 
 
 class GraphRepr(BaseRepr):
-    """This class presents a complete representation of a graph and its individual
-    subcomponents, including Operators, Dataframes, and Variables. Graph representations
-    are used during execution to load functions and pass data to the correct operators.
-    """
+    """This class presents a complete representation of a graph.
 
-    def __init__(self, name: str):
-        # TODO(Chiiizzzy)
+    A graph contains individual subcomponents, including Operators, Dataframes, and
+    Variables. Graph representations are used during execution to load functions and
+    pass data to the correct operators.
+
+    Args:
+        name(`str`):
+            The representation name.
+        file_or_url(`str`):
+            The file or remote url that stores the information of this representation.
+    """
+    def __init__(self, name: str, file_or_url: str = None):
```

Comment:
We may support local file/HTTP/HDFs or other file systems, so we can use file_str, check types, 
HTTP:  file_str.low().startwiths("http")
HDFS: file_str.low().startwiths("hdfs")
local: file_str.low().startwiths('/') and so on. Write a function to check  the type and read the  file

Reply:
changed